README.md

```diff
@@ -49,20 +49,9 @@ A Ruby blog app is a web application that allows users to create, edit, and publ
 
 > The key features of this project includes.
 
-- **Ability to catalog any type of item**
-- **Ability to add photos of items**
-- **Ability to view and edit item details**
-- **Ability to track item status**
-- **Ability to generate reports**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-## 🚀 Live Demo <a name="live-demo"></a>
-
-The Live Demo of this Capstone Project is available now on YouTube.
-
```

Comment:
- [ ] Please remove the live demo section from your project for now till you have a deployed link, this makes your readme more professional 👍🏿 


<img width="586" alt="test" src="https://github.com/SaveryIV/blog-app-on-Rails/assets/69638013/26537215-f582-4acd-b5d4-e73a7da98645">

- [ ] Kindly update your future features section, content it is still referring to my catatlog of things project 👍🏿 

<img width="934" alt="ttt" src="https://github.com/SaveryIV/blog-app-on-Rails/assets/69638013/56380612-79bd-4cf6-9107-20e250111981">